platform/util/base/src/com/intellij/openapi/util/NlsSafe.java

```diff
@@ -16,7 +16,7 @@
  *   <li>URL</li>
  *   <li>Programming language or framework name</li>
  * </ul>
- * Avoid using NlsSafe just to suppress the "hardcoded string" inspection warning. Use @{@link NonNls} 
+ * Avoid using NlsSafe just to suppress the "hardcoded string" inspection warning. Use {@link NonNls}
```

Comment:
For annotation, it looks like the `@` character was intended. Though it's probably better to put it inside the link. I will fix